src/lsm/schema.zig

```diff
@@ -509,3 +520,122 @@ pub const TableData = struct {
         return schema.block_values_bytes_const(data_block)[0..used_bytes];
     }
 };
+
+/// A ManifestLog block's body is a SoA of Labels and TableInfos.
+/// Each Label/TableInfo pair is an "entry".
+// TODO Store timestamp (snapshot) in header.
+pub const ManifestLog = struct {
+    const entry_size = @sizeOf(Label) + @sizeOf(TableInfo);
+    const entry_count_max_unaligned = @divFloor(block_body_size, entry_size);
+
+    pub const entry_count_max =
+        @divFloor(entry_count_max_unaligned, @alignOf(TableInfo)) * @alignOf(TableInfo);
+
+    const labels_size_max = entry_count_max * @sizeOf(Label);
+    const tables_size_max = entry_count_max * @sizeOf(TableInfo);
+
+    comptime {
+        assert(entry_count_max > 0);
+        assert(labels_size_max % @alignOf(TableInfo) == 0);
+        assert(tables_size_max % @alignOf(TableInfo) == 0);
+
+        // Bit 7 is reserved to indicate whether the event is an insert or remove.
+        assert(constants.lsm_levels <= std.math.maxInt(u7) + 1);
+
+        assert(@sizeOf(Label) == @sizeOf(u8));
+
+        // All TableInfo's should already be 16-byte aligned because of the leading checksum.
+        const alignment = 16;
```

Comment:
assert `alignment >= @sizeOf(vsr.Header)`.

Reply:
:thinking: Why `>=` instead of `alignment <= @sizeOf(vsr.Header)`? As in: "tables should not be aligned more strictly than the headers that offset them"?